AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -1471,6 +1472,7 @@ anekdoty.ru,championat.com,bloombergquint.com,kinogo-net-hd.ru,bigcinema-online.
 ! #%#//scriptlet("abort-on-property-read", "disableSelection")
 iembra2or.com,battle-one.com,terbit21.online,wizardsubs.com,audio-sound-premium.com,sysnettechsolutions.com,mustafabukulmez.com,ifdreamscametrue.com#%#//scriptlet("abort-on-property-read", "disableSelection")
 !
+toonnbook.nate.com#%#AG_onLoad(function() { setInterval(function() { document.querySelector('div#wrap')[Object.keys(document.querySelector('div#wrap'))[1]].onContextMenu = null; }, 1000); });
```

Comment:
It's better to use the scriptlet:
`#%#//scriptlet('prevent-addEventListener', 'contextmenu')`

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/95987/commits/0c747fbe7b09e9960475d04a29d14110d93ce5cd
Done.